<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Waiting for sink ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuCamera_views">
     <property name="title">
      <string>Camera views</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuCamera_performance">
     <property name="title">
      <string>Camera performance</string>
     </property>
    </widget>
    <addaction name="menuCamera_views"/>
    <addaction name="menuCamera_performance"/>
    <addaction name="actionView_camera_settings"/>
    <addaction name="actionView_network_topology"/>
    <addaction name="actionHide_main_window"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionViews">
   <property name="text">
    <string>Views</string>
   </property>
  </action>
  <action name="actionCamera_Settings">
   <property name="text">
    <string>Camera Settings</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionCamera_view_1">
   <property name="text">
    <string>Camera view 1</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="text">
    <string>Dummy</string>
   </property>
  </action>
  <action name="actionView_Camera_1">
   <property name="text">
    <string>view Camera 1</string>
   </property>
  </action>
  <action name="actionView_camera_settings">
   <property name="text">
    <string>Hide network topology</string>
   </property>
  </action>
  <action name="actionView_network_topology">
   <property name="text">
    <string>Hide network topology</string>
   </property>
  </action>
  <action name="actionHide_main_window">
   <property name="text">
    <string>Hide main window</string>
   </property>
  </action>
  <action name="actionAsd">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_operativeMode"/>
 </buttongroups>
</ui>
